macros/parsers/parserGraphTool.pl

```diff
@@ -610,27 +616,28 @@ sub constructJSXGraphOptions {
 }
 
 # Produce a hidden answer rule to contain the JavaScript result and insert the graphbox div and
-# javacript to display the graph tool.  If a hard copy is being generated, then PGgraphmacros.pl
-# is used to generate a printable graph instead.  An attempt is made to make the printable graph
-# look as much as possible like the JavaScript graph.
+# javacript to display the graph tool.  If a hard copy is being generated, then PGtikz.pl is used
```

Comment:
javacript -> javascript

Reply:
Actually, it should be `JavaScript`.  Thanks for pointing that out.  That was that way before.

Reply:
I fixed this typo in #791.  So no need to fix it here and cause more conflict with that pull request.